packages/app/package.json

```
@@ -96,7 +96,7 @@
     "@types/zen-observable": "^0.8.0",
     "axios": "^1.11.0",
     "cross-env": "^7.0.0",
-    "msw": "^1.0.0"
+    "msw": "^2.10.5"
```

```suggestion
    "msw": "^2.0.0"
```

Let's just use the same exact range globally in the whole repo; keeps our yarn.lock and customers' yarn.locks clean and easy to maintain